packages/ppx/test/css-support/transitions.t/run.t

```diff
@@ -55,4 +55,60 @@ This test only runs against Css_Js_Core from styled-ppx.css_native
   CssJs.transitionTimingFunction(`steps((5, `end_)));
   CssJs.transitionDelay(`s(1));
   CssJs.transitionDelay(`s(-1));
-  CssJs.unsafe({js|transition|js}, {js|1s 2s width linear|js});
+  CssJs.transitionList([|
+    CssJs.Transition.shorthand(
+      ~duration=?Some(`s(1)),
+      ~delay=?Some(`s(2)),
+      ~timingFunction=?Some(`linear),
+      {js|width|js},
+    ),
+  |]);
+  CssJs.transitionList([|`none|]);
+  CssJs.transitionList([|
+    CssJs.Transition.shorthand(
+      ~duration=?None,
+      ~delay=?None,
+      ~timingFunction=?None,
+      {js|margin-right|js},
+    ),
+  |]);
+  CssJs.transitionList([|
+    CssJs.Transition.shorthand(
+      ~duration=?None,
+      ~delay=?None,
+      ~timingFunction=?Some(`easeIn),
+      {js|margin-right|js},
+    ),
+  |]);
+  CssJs.transitionList([|
+    CssJs.Transition.shorthand(
+      ~duration=?Some(`ms(500)),
+      ~delay=?None,
+      ~timingFunction=?None,
+      {js|all|js},
+    ),
+  |]);
+  CssJs.transitionList([|
+    CssJs.Transition.shorthand(
+      ~duration=?Some(`ms(200)),
+      ~delay=?Some(`ms(500)),
+      ~timingFunction=?None,
+      {js|all|js},
+    ),
+  |]);
+  CssJs.transitionList([|
+    CssJs.Transition.shorthand(
+      ~duration=?None,
+      ~delay=?None,
+      ~timingFunction=?Some(`linear),
+      {js|all|js},
+    ),
+  |]);
+  CssJs.transitionList([|
+    CssJs.Transition.shorthand(
+      ~duration=?Some(`s(1)),
+      ~delay=?Some(`ms(500)),
+      ~timingFunction=?Some(`linear),
+      {js|margin-right|js},
+    ),
+  |]);
```

Comment:
Would be nice to add a few tests under css-support (packages/ppx/test/css-support) to make sure the interface (CssJs) type-checks and also a few tests cases with interpolation, which might be a bit tricky to get right since we use "||" for the spec.

Reply:
yup deffo needed

Reply:
added!